nixos/modules/programs/iay.nix

```
@@ -0,0 +1,38 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.iay;
+
+in {
+  options.programs.iay = {
+    enable = mkEnableOption (lib.mdDoc "the Starship shell prompt");
```

1. Starship???
2. Use the new constructions:
```suggestion
    enable = mkEnableOption (mdDoc "iay");
    package = mkPackageOption pkgs "iay" {};
```

---

Done.